domains/transfer/Transfer.API/API/Transfer/TransferAgreementProposalCleanup/TransferAgreementProposalCleanupService.cs

```diff
@@ -62,7 +62,7 @@ private async Task DeleteOldTransferAgreementProposals(DateTimeOffset olderThan,
 
         var activityLogEntries = oldProposals.Select(proposal => ActivityLogEntry.Create(Guid.Empty, ActivityLogEntry.ActorTypeEnum.System,
             string.Empty, string.Empty, string.Empty, ActivityLogEntry.EntityTypeEnum.TransferAgreementProposal,
-            ActivityLogEntry.ActionTypeEnum.Expired, proposal.Id));
+            ActivityLogEntry.ActionTypeEnum.Expired, proposal.Id.ToString()));
```

Comment:
I thought it was possible to cast the guid to a string, automatically?

Reply:
No no, you have to ToString() it

Reply:
ok